torch/csrc/jit/pybind_utils.h

```diff
@@ -67,6 +67,16 @@ inline Stack toStack(const py::tuple& inputs) {
   return toIValue(inputs).toTuple()->elements();
 }
 
+inline IValue toIValue(py::handle obj, const TypePtr& type);
+
+inline IValue createGenericList(py::handle obj, const TypePtr& elem_type) {
```

Comment:
Why is this called a "generic list" and not a tuple, like everywhere else? We don't really have a concept of an IValue list.

Reply:
Michael and I were discussing this: we have to have a fallback boxed list representation for lists of things that are not ints, float, or Tensors. This is just making this code robust to that future. Once Michaels mutable list patch lands, there will be a distinction between the Tuple in ivalue (const list), and the mutable boxed list type.